sqldelight-compiler/integration-tests/src/test/kotlin/com/example/TestDatabase.kt

```diff
@@ -12,23 +12,13 @@ interface TestDatabase : Transacter {
 
     val teamQueries: TeamQueries
 
-    object Schema : SqlDriver.Schema {
-        override val version: Int
-            get() = TestDatabaseImplExposer.schema.version
-
-        override fun create(driver: SqlDriver) = TestDatabaseImplExposer.schema.create(driver)
-
-        override fun migrate(
-            driver: SqlDriver,
-            oldVersion: Int,
-            newVersion: Int
-        ) = TestDatabaseImplExposer.schema.migrate(driver, oldVersion, newVersion)
-    }
-
     companion object {
+        val Schema: SqlDriver.Schema
```

Comment:
capitalized so that people's consuming code doesnt break. (This used to be `object Schema`)

Reply:
Hmm it's source compatible but it's binary incompatible. I think that's fine since it's generated code.

Reply:
yea